src/lib.rs

```diff
@@ -183,7 +87,12 @@ pub fn with_in_background<T: AsRef<OsStr> + Sized>(
 mod windows {
     use std::ffi::OsStr;
     use std::io;
-    use std::os::windows::ffi::OsStrExt;
+    use std::os::windows::{ffi::OsStrExt, process::ExitStatusExt};
+    use std::process::{Command, ExitStatus};
+    use std::ptr;
+
+    use winapi::ctypes::c_int;
+    use winapi::um::shellapi::ShellExecuteW;
 
     pub fn convert_path(path: &OsStr) -> io::Result<Vec<u16>> {
```

Comment:
I believe this function doesn't have to be `pub` anymore.

Reply:
fixed in 3507b55